services/compute/virtualmachine/virtualmachine.go

```diff
@@ -922,3 +929,26 @@ func (c *client) getZoneConfiguration(zoneConfiguration *wssdcommon.ZoneConfigur
 		StrictPlacement: &strinctPlacement,
 	}
 }
+
+func (c *client) getWssdPlacementGroupReference(s *compute.PlacementGroupReference) (*wssdcloudcompute.PlacementGroupReference, error) {
+	if s == nil {
+		return nil, nil
```

Comment:
in case of s is nil, we dont throw an error?

Reply:
discussed offline